src/modules/getLikesData.js

```diff
@@ -0,0 +1,9 @@
+const getLikesurl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/votSVyM8b966LHhFVfLV/likes';
+
+const getLikesData = async () => {
+  const response = await fetch(getLikesurl);
+  const feedback = await response.json();
+  return feedback;
```

Comment:
Here in this powerful function you wrote, I could see that, the function is fetching a resource from the API, but there is no id in the URL for the **likes** endpoint like this /apps/**:app_id**/likes/ The app_id represents the id of a resource you are fetching from the API. Please kindly have a second look to be sure. That may also be the reason why clicking on each like behaves undesirably like that.

Reply:
Thanks. The API has no add ID as that of the reservation and comments. It ends with 'likes' for both POST and GET. 👍